backend/src/main/java/io/debezium/configserver/rest/ConnectorResource.java

```diff
@@ -102,7 +103,7 @@ public Response getClusters() throws InvalidClusterException {
             responseCode = "200",
             content = @Content(
                     mediaType = MediaType.APPLICATION_JSON,
-                    schema = @Schema(implementation = ConnectorType.class)
+                    schema = @Schema(implementation = ConnectorProperty.class, type = SchemaType.ARRAY)
```

Comment:
Like in your task comment, I expected that we keep `ConnectorType.class` here. (See my email and let's discuss it in today's standup call).